.github/docker/linux_amd64/Dockerfile

```
@@ -0,0 +1,124 @@
+RUN "/usr/lib/llvm-${MAJOR}/bin" >> "$GITHUB_PATH"
+
+# Install cmake 3.31
+RUN mkdir /cmake_3_31 && \
+    cd /cmake_3_31 && \
+    wget https://github.com/Kitware/CMake/releases/download/v3.31.3/cmake-3.31.3-linux-x86_64.sh && \
+    chmod +x cmake-3.31.3-linux-x86_64.sh && \
+    ./cmake-3.31.3-linux-x86_64.sh --skip-license --prefix=/usr/local && \
+    cmake --version
+
+# Install GIT
+RUN wget https://github.com/git/git/archive/refs/tags/v2.25.0.tar.gz && \
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)